mk/platform.mk.macosx

```diff
@@ -6,9 +6,13 @@ LIB_PREFIX := lib
 
 LIB_EXT := .a
 
+ifeq ($(origin CXX),default)

```

Comment:
`CC` and `CXX` variables have a default value, which is `gcc` and `g++` respectively. This check is necessary because we need to know if the value assigned to that is the default, or has been user provided by overriding the environment variables.

Reply:
ok that makes sesne